qemu/tests/cpu_device_hotplug_during_boot.py

```diff
@@ -21,14 +23,16 @@ def run(test, params, env):
     """
     vcpu_devices = params.objects("vcpu_devices")
     unplug_during_boot = params.get_boolean("unplug_during_boot")
+    vcpus_count = params.get_numeric("vcpus_count", 1)
+    pluggable_count = len(vcpu_devices) * vcpus_count
     boot_patterns = [r".*Started udev Wait for Complete Device Initialization.*"]
     reboot_patterns = [r".*[Rr]ebooting.*", r".*[Rr]estarting system.*",
                        r".*[Mm]achine restart.*"]
 
     vm = env.get_vm(params["main_vm"])
     vm.verify_alive()
     smp = vm.cpuinfo.smp
-    maxcpus = vm.cpuinfo.maxcpus
+    expected_count = pluggable_count + smp
```

Comment:
Suggest `expected_count = enabled_cpu + smp` after L45.

Reply:
They are equal. If anyone CPU device failed to hotplug, the case will terminate.

Reply:
If cpu device hotplug failed, then will not check cpu count, there is no point to get expected_count.

Reply:
So how to get the enabled_cpu? The count is equal to pluggable_count.

Reply:
It's same as in check_if_vm_initial_vcpu_match.
```
if vcpu_params.get_boolean("vcpu_enable"):
            enabled_vcpu_devices.append(vcpu_device)
expected_count = vm.cpuinfo.smp + (len(enabled_vcpu_devices) * vcpus_count)
```

Reply:
Whether a vcpu device is successfully plugged or not, `vcpu_enable` will not be changed in params.

Reply:
Hmm, I think I need to do a test to confirm this one.

Reply:
I think I got wrong, `params` would not be changed after vcpu hotplug/hotunplug, but `vm.params` would.
So `vm.params` can be used, updated patch accordingly.